reana/cli.py

```diff
@@ -1695,6 +1721,66 @@ def python_install_eggs():
             run_command(cmd, component)
 
 
+@cli.command(name='python-unit-tests')
+@click.option('--component', '-c', multiple=True, default=['ALL'],
+              help='Which components? [shortname|name|.|CLUSTER|ALL]')
+def python_unit_tests(component):  # noqa: D301
+    """Run Python unit tests in independent environments.
+
+    For each component, create a dedicated throw-away virtual environment,
+    install latest shared modules (reana-commons, reana-db) that are currently
+    checked-out and run the usual component unit tests. Delete the throw-away
+    virtual environment afterwards.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
```

Comment:
On line 1725 I see that default is `'ALL'`

Reply:
Good catch, taken note, now that the branch is merged, let's push a fix for both comments separately